test/Makefile

```diff
@@ -1,13 +1,14 @@
 SRCDIR := $(abspath $(dir $(lastword $(MAKEFILE_LIST))))
 JULIAHOME := $(abspath $(SRCDIR)/..)
 BUILDDIR := .
-STDLIBDIR := $(abspath $(JULIAHOME)/stdlib)
 include $(JULIAHOME)/Make.inc
+VERSDIR := v$(shell cut -d. -f1-2 < $(JULIAHOME)/VERSION)
```

Comment:
aka `v$(SOMAJOR)`?

Reply:
ah, I see this is what we do in `Makefile`. carry on.

Reply:
Yea, just copied it from there.